Essentials/src/com/earth2me/essentials/commands/Commandtpaall.java

```diff
@@ -30,7 +31,7 @@ public void run(final Server server, final CommandSource sender, final String co
     }
 
     private void teleportAAllPlayers(final Server server, final CommandSource sender, final User target) {
-        sender.sendMessage(tl("teleportAAll"));
+        sender.sendMessage(tl("teleportAAll")); // TODO Take a look at whether we want to send this message before the loop, due to the TPARequestEvent implementation
```

Comment:
Is this TODO still applicable? I think the initial message should be sent regardless of whether some are disabled.